vic/vic_run/src/surface_fluxes.c

```diff
@@ -797,49 +793,17 @@ surface_fluxes(bool                 overstory,
         /**************************************
            Compute Potential Evap
         **************************************/
-        // First, determine the stability correction used in the iteration
-        if (iter_aero_resist_used[0] == param.HUGE_RESIST) {
-            stability_factor[0] = param.HUGE_RESIST;
-        }
-        else {
-            stability_factor[0] = iter_aero_resist_used[0] /
-                                  aero_resist[N_PET_TYPES][UnderStory];
-        }
-        if (iter_aero_resist_used[1] == iter_aero_resist_used[0]) {
-            stability_factor[1] = stability_factor[0];
-        }
-        else {
-            if (iter_aero_resist_used[1] == param.HUGE_RESIST) {
-                stability_factor[1] = param.HUGE_RESIST;
-            }
-            else {
-                stability_factor[1] = iter_aero_resist_used[1] /
-                                      aero_resist[N_PET_TYPES][1];
-            }
-        }
 
-        // Next, loop over pot_evap types and apply the correction to the relevant aerodynamic resistance
-        for (p = 0; p < N_PET_TYPES; p++) {
-            if (stability_factor[0] == param.HUGE_RESIST) {
-                step_aero_resist[p][0] = param.HUGE_RESIST;
-            }
-            else {
-                step_aero_resist[p][0] = aero_resist[p][UnderStory] *
-                                         stability_factor[0];
-            }
-            if (stability_factor[1] == param.HUGE_RESIST) {
-                step_aero_resist[p][1] = param.HUGE_RESIST;
-            }
-            else {
-                step_aero_resist[p][1] = aero_resist[p][1] *
-                                         stability_factor[1];
-            }
-        }
-
-        // Finally, compute pot_evap
-        compute_pot_evap(veg_class, dmy, gp->dt, atmos->shortwave[hidx],
-                         iter_soil_energy.NetLongAtmos, Tair, VPDcanopy,
-                         soil_con->elevation, step_aero_resist, iter_pot_evap);
+        compute_pot_evap(gp->dt, vic_run_veg_lib[veg_class].rmin,
```

Comment:
I don't think you need to pass in `gp->dt` since that is available as an `extern`.  It is also better to use `STEPS_PER_DAY`.